sdk/agent/agent.go

```diff
@@ -276,6 +276,15 @@ func (a *Agent) Open() error {
 // The payment is not authorized until the remote participant signs the payment
 // and returns the payment.
 func (a *Agent) Payment(paymentAmount int64) error {
+	return a.PaymentWithMemo(paymentAmount, "")
+}
+
+// Payment makes a payment of the payment amount to the remote participant using
```

Comment:
💡 can we remove the dup method comment from `Payment()` above? Since we're re-explaining what the method does here. The Payment comment above could be something like:
```
// Payment creates a payment using the PaymentWithmemo method with an empty memo.
func (a *Agent) Payment(paymentAmount int64) error {
```

Reply:
#351